airflow/config_templates/config.yml

```diff
@@ -1130,6 +1130,56 @@ metrics:
       type: string
       example: ~
       default: "False"
+traces:
+  description: |
+    Distributed traces integration settings.
+  options:
+    otel_on:
+      description: |
+        Enables sending traces to OpenTelemetry.
+      version_added: 2.8.2
```

Comment:
```suggestion
      version_added: 2.9.0
```

Here and all other places.

Reply:
This is a new feature and we will add it in 2.9.0

Reply:
Ok, thanks for letting me know the correct version.